src/es/cuevana/src/eu/kanade/tachiyomi/animeextension/es/cuevana/Cuevana.kt

```diff
@@ -52,15 +52,15 @@ class Cuevana : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
         Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
     }
 
-    override fun popularAnimeSelector(): String = "section li.xxx.TPostMv div.TPost"
+    override fun popularAnimeSelector(): String = ".MovieList .TPostMv .TPost"
 
-    override fun popularAnimeRequest(page: Int): Request = GET("$baseUrl/peliculas/page/$page")
+    override fun popularAnimeRequest(page: Int): Request = GET("$baseUrl/peliculas?page=$page")
 
     override fun popularAnimeFromElement(element: Element): SAnime {
         val anime = SAnime.create()
-        anime.setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
+        anime.setUrlWithoutDomain(baseUrl + element.selectFirst("a")!!.attr("href"))
```

Comment:
You dont need to prepend the baseUrl when using `setUrlWithoutDomain`, since that function is going to remove the domain